bzl/workspace.bzl

```diff
@@ -90,8 +90,8 @@ def plaidml_workspace():
         strip_prefix = "jsonnet-0.13.0",
     )
 
-    LLVM_COMMIT = "a38882d8d63e9e093926789331b6c5b4711a889e"
-    LLVM_SHA256 = "8f5dd8dc9479eb7fae76376947c0155f800bc21a508454645d2563024102467e"
+    LLVM_COMMIT = "8e46cd4d6713a1775672a148bd8c7c20b8badb62"
```

Comment:
This appears to be one commit behind the PR that landed there.

But the bigger issue is that we need to pull in the latest from upstream master into the `plaidml/plaidml-v1` branch of the `plaidml/llvm-project` repo.

Reply:
Moved to the latest plaidml/llvm-project.
As we discussed, we will move to the latest LLV M upstream when the upstream changes for vectorization land.

Reply:
Also, don't get rid of the SHA!

Reply:
Oh wait, nevermind... missed it.